### PR TITLE
use MSbuild Copy

### PR DESCRIPTION
**Pull request by SimonCropp**
#skip-changelog

### CODE REVIEW

**Review by bruno-garcia — APPROVED** [review]

File: Directory.Build.targets [hunk 1]
```diff
@@ -200,7 +200,9 @@ Expected to exist:
     </ItemGroup>
 
     <Copy SourceFiles="@(AndroidSdkArtifacts)" DestinationFiles="@(AndroidSdkArtifacts->'$(SentryAndroidArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Exec WorkingDirectory="$(SentryAndroidRoot)" Command="cp sentry/build/libs/sentry*.jar $(SentryAndroidArtifactsDestination)sentry.jar" />
+
+    <Copy SourceFiles="$(SentryAndroidRoot)/sentry/build/libs/sentry*.jar"
```
Comment (bruno-garcia) [inline Directory.Build.targets:204]: Wildcards don't work with that task do it?

Comment (SimonCropp) [reply Directory.Build.targets:204]: do we need the wildcard? is the file name not known?

Comment (vaind) [reply Directory.Build.targets:204]: The source name contains a version number while we want the target without that.

**Review by vaind — CHANGES_REQUESTED** [review]
The changes are not just cosmetic but change the behaviour. I'm also missing the reason why this needs to be changed. It currently runs fine on all 3 desktop platforms in either powershell or bash/zsh. If we know whats the issue maybe we can find a solution that doesn't break the rest/changes the expected behaviour

**Comment by SimonCropp** [comment]
@vaind  `cp` wont run on windows

**Comment by vaind** [comment]
> 

I'm on windows and yes it does, you just have to use powershell instead of batch

**Comment by SimonCropp** [comment]
forme, i am in Rider. and the build fails

**Pull request closed without merging**